jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/spinner/SpinnerButtonsTests.java

```diff
@@ -0,0 +1,95 @@
+package io.github.epam.bootstrap.tests.composite.section.spinner;
+
+import com.epam.jdi.light.driver.WebDriverFactory;
+import com.epam.jdi.light.ui.bootstrap.elements.complex.ButtonWithSpinner;
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import static com.epam.jdi.light.elements.init.UIFactory.$;
+import static io.github.com.StaticSite.bsPage;
+import static io.github.com.pages.BootstrapPage.buttonWithGrowingSpinner;
+import static io.github.com.pages.BootstrapPage.buttonWithGrowingSpinnerAndText;
+import static io.github.com.pages.BootstrapPage.buttonWithSpinner;
+import static io.github.com.pages.BootstrapPage.buttonWithSpinnerAndText;
+import static io.github.epam.states.States.shouldBeLoggedIn;
+import static org.hamcrest.CoreMatchers.is;
+import static org.testng.Assert.assertNotEquals;
+
+public class SpinnerButtonsTests extends TestsInit {
+
+    private final String buttonText = "Loading...";
+    private final String spinnerClass = "spinner-border";
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+    }
+
+    @DataProvider
+    public Object[][] buttonsWithGrowingSpinners() {
+        return new Object[][]{
+                {buttonWithGrowingSpinner},
+                {buttonWithGrowingSpinnerAndText},
+        };
+    }
+
+    @Test()
+    public void checkButtonText() {
+        buttonWithSpinnerAndText.isDisplayed();
```

Comment:
buttonWithSpinnerAndText.isDisplayed(); - it just returns true or false, it's not a check

Reply:
fixed